api/_lib/util/error-handler.ts

```diff
@@ -13,6 +13,7 @@ function initSentry() {
   sentryInitDone = true;
 
   init({
+    // @ts-ignore
```

Comment:
**suggestion (non-blocking):** add comment about why

Can you add a comment here about why this was ignored?